src/Accelerators/Accelerator.hpp

```diff
@@ -51,6 +51,10 @@
 
 #define ACCEL_PROFILEIR_CL_ENUM(name) PROFILEIR_CL_ENUM_##name
 
+#define ACCEL_OPTREPORT_ENUM(name) OPTREPORT_ENUM_##name
+
+#define ACCEL_OPTREPORT_CL_ENUM(name) OPTREPORT_CL_ENUM_##name
```

Comment:
Could you update [this document](https://github.com/onnx/onnx-mlir/blob/main/docs/AddCustomAccelerators.md#21-macro) to include the new macro? Thanks!